### ISSUE TITLE
globalPosition returned by GestureDragUpdateCallback is compromised when app is suspended

### ISSUE THREAD

**Issue by miguelcmedeiros** [flutter/flutter#24665]
## Steps to Reproduce
In the following widget, if I start a drag gesture and if I suspend the app (like receiving a phone call) then the value of globalPosition is totally random.

```dart
class DemoScreen extends StatelessWidget {
  @override
  Widget build(BuildContext context) => Scaffold(
        body: GestureDetector(
          onPanUpdate: (details) => print(details.globalPosition),
          child: Container(
            color: Colors.orange,
          ),
        ),
      );
}
```

**Log**

```
flutter: Offset(96.3, 374.0)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(96.7, 374.3)
flutter: Offset(-50.8, 1422.5) -> when app is suspended
```

**Flutter doctor**
```
flutter doctor -v
[✓] Flutter (Channel unknown, v0.10.2, on Mac OS X 10.13.6 17G65, locale en-NL)
    • Flutter version 0.10.2 at /Users/user/IdeaProjects/hue_blue/flutter
    • Framework revision d8cbb80206 (4 weeks ago), 2018-10-26 01:30:21 -0400
    • Engine revision 6c2ade9fa2
    • Dart version 2.1.0-dev.8.0.flutter-bf26f760b1

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.io/setup/#android-setup for detailed instructions).
      If Android SDK has been installed to a custom location, set $ANDROID_HOME to that location.
      You may also want to add it to your PATH environment variable.


[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 2.0.0
    • CocoaPods version 1.5.3

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.io/setup/#android-setup for detailed instructions).

[✓] IntelliJ IDEA Community Edition (version 2018.1.6)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 30.0.1
    • Dart plugin version 181.4892.1

[✓] Connected device (1 available)
    • iPhone XS • B74C2265-233F-4EAB-89B5-6850589648EF • ios • iOS 12.1 (simulator)

! Doctor found issues in 2 categories.
```

**Comment by iapicca** [comment]
Hi @miguelcmedeiros 
I tested your code with the latest master
and the problem appear to be fixed

<details>
  <summary>logs</summary>

```bash 
[ +112 ms] Syncing files to device Pixel 3a...
[   +2 ms] Scanning asset files
[   +2 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +1 ms] <- recompile package:master/main.dart d985db91-6022-4fae-aee2-7c4a68a20fdc
[        ] <- d985db91-6022-4fae-aee2-7c4a68a20fdc
[  +84 ms] Updating files
[ +134 ms] DevFS: Sync finished
[   +7 ms] Syncing files to device Pixel 3a... (completed in 226ms)
[   +1 ms] Synced 0.8MB.
[  +13 ms] <- accept
[        ] Connected to _flutterView/0x7cbc4c4820.
[   +2 ms] Flutter run key commands.
[   +2 ms] r Hot reload. 🔥🔥🔥
[   +1 ms] R Hot restart.
[        ] h Repeat this help message.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on Pixel 3a is available at: http://127.0.0.1:59626/S7GuXlhDzBY=/
[+2762 ms] I/flutter ( 4616): Offset(300.7, 342.9)
[        ] I/flutter ( 4616): Offset(294.9, 344.4)
[   +1 ms] I/flutter ( 4616): Offset(285.8, 346.9)
[        ] I/flutter ( 4616): Offset(275.6, 350.2)
[        ] I/flutter ( 4616): Offset(264.0, 355.3)
[        ] I/flutter ( 4616): Offset(251.6, 360.7)
[        ] I/flutter ( 4616): Offset(238.9, 367.3)
[        ] I/flutter ( 4616): Offset(226.9, 373.8)
[        ] I/flutter ( 4616): Offset(213.1, 382.2)
[        ] I/flutter ( 4616): Offset(203.6, 390.2)
[   +1 ms] I/flutter ( 4616): Offset(193.5, 398.2)
[   +9 ms] I/flutter ( 4616): Offset(183.6, 406.5)
[   +8 ms] I/flutter ( 4616): Offset(176.4, 414.5)
[   +9 ms] I/flutter ( 4616): Offset(169.5, 422.2)
[   +7 ms] I/flutter ( 4616): Offset(162.2, 429.5)
[   +9 ms] I/flutter ( 4616): Offset(154.9, 438.2)
[   +7 ms] I/flutter ( 4616): Offset(149.1, 446.9)
[  +10 ms] I/flutter ( 4616): Offset(144.7, 453.5)
[   +8 ms] I/flutter ( 4616): Offset(140.7, 461.1)
[   +9 ms] I/flutter ( 4616): Offset(137.5, 468.0)
[   +7 ms] I/flutter ( 4616): Offset(135.6, 472.7)
[   +8 ms] I/flutter ( 4616): Offset(134.9, 475.6)
[   +8 ms] I/flutter ( 4616): Offset(134.5, 477.1)
[  +27 ms] I/flutter ( 4616): Offset(135.6, 477.5)
[   +7 ms] I/flutter ( 4616): Offset(141.5, 477.1)
[   +7 ms] I/flutter ( 4616): Offset(150.5, 474.9)
[   +8 ms] I/flutter ( 4616): Offset(158.9, 472.7)
[   +8 ms] I/flutter ( 4616): Offset(166.9, 470.2)
[   +8 ms] I/flutter ( 4616): Offset(178.5, 466.9)
[   +8 ms] I/flutter ( 4616): Offset(187.6, 464.0)
[   +7 ms] I/flutter ( 4616): Offset(199.3, 460.7)
[   +8 ms] I/flutter ( 4616): Offset(211.6, 456.0)
[   +8 ms] I/flutter ( 4616): Offset(221.5, 449.8)
[   +8 ms] I/flutter ( 4616): Offset(233.5, 441.8)
[  +10 ms] I/flutter ( 4616): Offset(242.2, 435.3)
[   +8 ms] I/flutter ( 4616): Offset(249.1, 428.4)
[   +9 ms] I/flutter ( 4616): Offset(255.3, 421.1)
[   +9 ms] I/flutter ( 4616): Offset(260.4, 414.9)
[   +8 ms] I/flutter ( 4616): Offset(264.7, 409.5)
[   +8 ms] I/flutter ( 4616): Offset(269.1, 405.1)
[   +6 ms] I/flutter ( 4616): Offset(273.8, 400.4)
[   +9 ms] I/flutter ( 4616): Offset(278.5, 396.7)
[   +8 ms] I/flutter ( 4616): Offset(282.2, 393.5)
[   +9 ms] I/flutter ( 4616): Offset(285.5, 391.3)
[   +8 ms] I/flutter ( 4616): Offset(288.0, 388.7)
[   +8 ms] I/flutter ( 4616): Offset(290.2, 386.9)
[   +8 ms] I/flutter ( 4616): Offset(291.6, 385.8)
[   +9 ms] I/flutter ( 4616): Offset(293.1, 384.0)
[   +8 ms] I/flutter ( 4616): Offset(293.5, 382.9)
[   +8 ms] I/flutter ( 4616): Offset(293.8, 381.8)
[   +8 ms] I/flutter ( 4616): Offset(293.5, 381.5)
[   +8 ms] I/flutter ( 4616): Offset(288.7, 379.6)
[   +9 ms] I/flutter ( 4616): Offset(280.7, 380.4)
[   +6 ms] I/flutter ( 4616): Offset(269.1, 384.4)
[  +10 ms] I/flutter ( 4616): Offset(257.1, 390.5)
[   +7 ms] I/flutter ( 4616): Offset(242.9, 397.5)
[   +9 ms] I/flutter ( 4616): Offset(230.9, 403.6)
[   +7 ms] I/flutter ( 4616): Offset(219.3, 410.5)
[  +10 ms] I/flutter ( 4616): Offset(209.5, 417.5)
[  +11 ms] I/flutter ( 4616): Offset(202.5, 422.5)
[   +5 ms] I/flutter ( 4616): Offset(197.1, 426.5)
[   +8 ms] I/flutter ( 4616): Offset(193.5, 429.8)
[   +8 ms] I/flutter ( 4616): Offset(190.9, 432.0)
[   +8 ms] I/flutter ( 4616): Offset(188.0, 435.3)
[   +8 ms] I/flutter ( 4616): Offset(185.5, 438.5)
[  +10 ms] I/flutter ( 4616): Offset(182.5, 442.5)
[   +7 ms] I/flutter ( 4616): Offset(180.4, 446.5)
[   +7 ms] I/flutter ( 4616): Offset(178.9, 450.2)
[  +14 ms] I/flutter ( 4616): Offset(177.5, 453.5)
[   +3 ms] I/flutter ( 4616): Offset(176.0, 456.7)
[   +9 ms] I/flutter ( 4616): Offset(174.9, 460.0)
[   +8 ms] I/flutter ( 4616): Offset(174.2, 464.0)
[   +8 ms] I/flutter ( 4616): Offset(173.5, 467.3)
[  +11 ms] I/flutter ( 4616): Offset(173.5, 469.8)
[   +4 ms] I/flutter ( 4616): Offset(173.8, 471.6)
[   +8 ms] I/flutter ( 4616): Offset(174.9, 473.1)
[   +8 ms] I/flutter ( 4616): Offset(177.5, 475.6)
[   +9 ms] I/flutter ( 4616): Offset(183.3, 478.2)
[   +8 ms] I/flutter ( 4616): Offset(188.4, 479.3)
[   +9 ms] I/flutter ( 4616): Offset(192.4, 480.0)
[   +9 ms] I/flutter ( 4616): Offset(196.7, 480.7)
[   +7 ms] I/flutter ( 4616): Offset(202.2, 480.7)
[   +8 ms] I/flutter ( 4616): Offset(208.0, 480.0)
[   +9 ms] I/flutter ( 4616): Offset(213.5, 478.5)
[   +8 ms] I/flutter ( 4616): Offset(218.5, 475.6)
[   +8 ms] I/flutter ( 4616): Offset(225.5, 471.3)
[   +7 ms] I/flutter ( 4616): Offset(232.0, 466.9)
[  +10 ms] I/flutter ( 4616): Offset(237.5, 463.3)
[   +8 ms] I/flutter ( 4616): Offset(241.8, 459.6)
[   +9 ms] I/flutter ( 4616): Offset(245.1, 457.1)
[   +8 ms] I/flutter ( 4616): Offset(248.0, 453.8)
[   +8 ms] I/flutter ( 4616): Offset(252.0, 449.8)
[   +9 ms] I/flutter ( 4616): Offset(256.4, 445.5)
[   +7 ms] I/flutter ( 4616): Offset(260.4, 441.5)
[   +9 ms] I/flutter ( 4616): Offset(264.4, 437.8)
[   +9 ms] I/flutter ( 4616): Offset(268.0, 434.2)
[   +7 ms] I/flutter ( 4616): Offset(272.4, 429.1)
[   +9 ms] I/flutter ( 4616): Offset(276.4, 424.0)
[   +8 ms] I/flutter ( 4616): Offset(279.6, 418.5)
[   +8 ms] I/flutter ( 4616): Offset(282.5, 414.2)
[   +8 ms] I/flutter ( 4616): Offset(285.1, 410.9)
[   +8 ms] I/flutter ( 4616): Offset(286.9, 407.6)
[   +8 ms] I/flutter ( 4616): Offset(288.7, 405.1)
[   +9 ms] I/flutter ( 4616): Offset(290.2, 402.5)
[   +6 ms] I/flutter ( 4616): Offset(291.6, 400.0)
[   +9 ms] I/flutter ( 4616): Offset(292.4, 398.2)
[  +10 ms] I/flutter ( 4616): Offset(292.4, 396.4)
[   +8 ms] I/flutter ( 4616): Offset(292.4, 394.2)
[   +6 ms] I/flutter ( 4616): Offset(291.6, 393.1)
[   +8 ms] I/flutter ( 4616): Offset(288.7, 390.9)
[   +8 ms] I/flutter ( 4616): Offset(283.6, 388.7)
[   +7 ms] I/flutter ( 4616): Offset(277.1, 388.0)
[   +8 ms] I/flutter ( 4616): Offset(266.5, 389.5)
[   +7 ms] I/flutter ( 4616): Offset(257.5, 391.6)
[   +9 ms] I/flutter ( 4616): Offset(246.9, 395.3)
[   +9 ms] I/flutter ( 4616): Offset(236.4, 398.5)
[   +8 ms] I/flutter ( 4616): Offset(228.0, 401.8)
[   +8 ms] I/flutter ( 4616): Offset(220.7, 404.4)
[   +8 ms] I/flutter ( 4616): Offset(212.4, 407.6)
[  +10 ms] I/flutter ( 4616): Offset(206.5, 410.2)
[   +8 ms] I/flutter ( 4616): Offset(201.5, 412.7)
[   +7 ms] I/flutter ( 4616): Offset(196.7, 416.0)
[   +8 ms] I/flutter ( 4616): Offset(191.3, 418.9)
[   +8 ms] I/flutter ( 4616): Offset(185.8, 421.8)
[  +10 ms] I/flutter ( 4616): Offset(182.2, 424.0)
[   +6 ms] I/flutter ( 4616): Offset(178.5, 426.2)
[   +9 ms] I/flutter ( 4616): Offset(175.6, 427.6)
[   +7 ms] I/flutter ( 4616): Offset(172.7, 429.8)
[   +8 ms] I/flutter ( 4616): Offset(170.5, 431.6)
[   +7 ms] I/flutter ( 4616): Offset(167.6, 434.2)
[   +8 ms] I/flutter ( 4616): Offset(165.8, 436.0)
[   +9 ms] I/flutter ( 4616): Offset(163.3, 438.5)
[   +7 ms] I/flutter ( 4616): Offset(161.5, 441.1)
[   +8 ms] I/flutter ( 4616): Offset(158.9, 443.6)
[   +8 ms] I/flutter ( 4616): Offset(157.1, 446.5)
[   +8 ms] I/flutter ( 4616): Offset(154.5, 449.5)
[   +8 ms] I/flutter ( 4616): Offset(152.7, 452.4)
[  +11 ms] I/flutter ( 4616): Offset(150.9, 454.9)
[   +5 ms] I/flutter ( 4616): Offset(149.5, 457.1)
[   +9 ms] I/flutter ( 4616): Offset(148.7, 458.9)
[   +7 ms] I/flutter ( 4616): Offset(148.7, 459.6)
[  +28 ms] I/flutter ( 4616): Offset(148.7, 459.6)
[   +1 ms] I/flutter ( 4616): Offset(151.3, 460.7)
[   +4 ms] I/flutter ( 4616): Offset(157.8, 461.5)
[   +9 ms] I/flutter ( 4616): Offset(163.3, 460.7)
[   +8 ms] I/flutter ( 4616): Offset(169.5, 459.6)
[   +7 ms] I/flutter ( 4616): Offset(177.8, 458.5)
[   +8 ms] I/flutter ( 4616): Offset(184.4, 457.1)
[   +8 ms] I/flutter ( 4616): Offset(190.9, 455.3)
[  +10 ms] I/flutter ( 4616): Offset(198.2, 452.7)
[   +6 ms] I/flutter ( 4616): Offset(208.0, 448.4)
[   +8 ms] I/flutter ( 4616): Offset(216.0, 443.6)
[   +7 ms] I/flutter ( 4616): Offset(225.5, 437.8)
[   +8 ms] I/flutter ( 4616): Offset(234.9, 432.4)
[   +9 ms] I/flutter ( 4616): Offset(241.8, 428.0)
[   +8 ms] I/flutter ( 4616): Offset(247.3, 424.0)
[   +8 ms] I/flutter ( 4616): Offset(252.0, 420.4)
[   +8 ms] I/flutter ( 4616): Offset(256.4, 417.1)
[   +8 ms] I/flutter ( 4616): Offset(260.4, 413.8)
[   +8 ms] I/flutter ( 4616): Offset(264.7, 411.3)
[   +8 ms] I/flutter ( 4616): Offset(268.4, 408.7)
[  +12 ms] I/flutter ( 4616): Offset(271.6, 406.2)
[   +4 ms] I/flutter ( 4616): Offset(275.6, 403.6)
[   +8 ms] I/flutter ( 4616): Offset(279.6, 400.7)
[   +8 ms] I/flutter ( 4616): Offset(283.3, 397.5)
[   +7 ms] I/flutter ( 4616): Offset(286.2, 394.9)
[   +9 ms] I/flutter ( 4616): Offset(289.1, 392.0)
[   +7 ms] I/flutter ( 4616): Offset(291.6, 389.1)
[   +9 ms] I/flutter ( 4616): Offset(294.5, 386.2)
[   +8 ms] I/flutter ( 4616): Offset(296.7, 383.6)
[   +7 ms] I/flutter ( 4616): Offset(299.6, 380.7)
[   +9 ms] I/flutter ( 4616): Offset(302.2, 377.8)
[   +8 ms] I/flutter ( 4616): Offset(304.7, 375.3)
[   +8 ms] I/flutter ( 4616): Offset(306.5, 373.1)
[   +7 ms] I/flutter ( 4616): Offset(308.4, 370.5)
[   +8 ms] I/flutter ( 4616): Offset(309.8, 368.7)
[   +9 ms] I/flutter ( 4616): Offset(310.9, 366.5)
[   +8 ms] I/flutter ( 4616): Offset(311.6, 364.7)
[   +8 ms] I/flutter ( 4616): Offset(311.6, 362.9)
[   +7 ms] I/flutter ( 4616): Offset(310.9, 361.5)
[  +10 ms] I/flutter ( 4616): Offset(309.5, 360.4)
[   +9 ms] I/flutter ( 4616): Offset(304.7, 357.5)
[   +5 ms] I/flutter ( 4616): Offset(298.5, 356.4)
[   +7 ms] I/flutter ( 4616): Offset(289.8, 356.7)
[   +8 ms] I/flutter ( 4616): Offset(282.5, 357.8)
[   +9 ms] I/flutter ( 4616): Offset(273.5, 360.0)
[   +9 ms] I/flutter ( 4616): Offset(262.2, 362.5)
[   +8 ms] I/flutter ( 4616): Offset(254.2, 364.7)
[   +8 ms] I/flutter ( 4616): Offset(244.0, 368.0)
[   +8 ms] I/flutter ( 4616): Offset(234.2, 370.9)
[   +9 ms] I/flutter ( 4616): Offset(226.2, 373.5)
[   +7 ms] I/flutter ( 4616): Offset(217.5, 376.0)
[   +8 ms] I/flutter ( 4616): Offset(207.6, 378.9)
[   +8 ms] I/flutter ( 4616): Offset(200.7, 381.8)
[   +8 ms] I/flutter ( 4616): Offset(193.8, 384.4)
[   +9 ms] I/flutter ( 4616): Offset(186.5, 387.3)
[   +8 ms] I/flutter ( 4616): Offset(179.3, 389.8)
[   +8 ms] I/flutter ( 4616): Offset(173.8, 392.4)
[   +8 ms] I/flutter ( 4616): Offset(168.0, 394.9)
[   +9 ms] I/flutter ( 4616): Offset(161.8, 398.5)
[   +7 ms] I/flutter ( 4616): Offset(155.6, 401.8)
[  +10 ms] I/flutter ( 4616): Offset(150.5, 405.5)
[   +8 ms] I/flutter ( 4616): Offset(145.8, 409.1)
[   +8 ms] I/flutter ( 4616): Offset(141.1, 412.4)
[   +8 ms] I/flutter ( 4616): Offset(136.7, 415.3)
[  +10 ms] I/flutter ( 4616): Offset(132.4, 418.5)
[   +7 ms] I/flutter ( 4616): Offset(128.0, 421.5)
[   +9 ms] I/flutter ( 4616): Offset(124.4, 424.4)
[   +8 ms] I/flutter ( 4616): Offset(121.1, 427.3)
[   +9 ms] I/flutter ( 4616): Offset(117.8, 430.2)
[   +8 ms] I/flutter ( 4616): Offset(115.3, 433.1)
[   +8 ms] I/flutter ( 4616): Offset(112.4, 435.6)
[   +8 ms] I/flutter ( 4616): Offset(110.2, 437.8)
[   +8 ms] I/flutter ( 4616): Offset(108.4, 439.6)
[   +9 ms] I/flutter ( 4616): Offset(106.9, 440.7)
[   +7 ms] I/flutter ( 4616): Offset(104.7, 442.2)
[   +8 ms] I/flutter ( 4616): Offset(103.6, 443.3)
[   +8 ms] I/flutter ( 4616): Offset(102.5, 444.0)
[   +8 ms] I/flutter ( 4616): Offset(101.5, 445.1)
[  +17 ms] I/flutter ( 4616): Offset(100.4, 445.5)
[   +1 ms] I/flutter ( 4616): Offset(99.6, 446.2)
[   +7 ms] I/flutter ( 4616): Offset(99.3, 446.2)
[   +9 ms] I/flutter ( 4616): Offset(99.3, 446.5)
[ +104 ms] I/flutter ( 4616): Offset(99.6, 445.8)
[        ] I/flutter ( 4616): Offset(103.6, 441.5)
[        ] I/flutter ( 4616): Offset(109.5, 435.3)
[        ] I/flutter ( 4616): Offset(115.3, 429.8)
[        ] I/flutter ( 4616): Offset(123.3, 424.0)
[        ] I/flutter ( 4616): Offset(131.3, 418.9)
[        ] I/flutter ( 4616): Offset(138.9, 414.2)
[        ] I/flutter ( 4616): Offset(147.6, 409.5)
[   +1 ms] I/flutter ( 4616): Offset(157.1, 405.1)
[   +9 ms] I/flutter ( 4616): Offset(164.7, 401.8)
[   +7 ms] I/flutter ( 4616): Offset(174.2, 398.2)
[   +9 ms] I/flutter ( 4616): Offset(183.6, 395.6)
[   +9 ms] I/flutter ( 4616): Offset(191.3, 393.5)
[   +7 ms] I/flutter ( 4616): Offset(201.1, 391.6)
[   +9 ms] I/flutter ( 4616): Offset(210.9, 389.8)
[   +8 ms] I/flutter ( 4616): Offset(218.5, 388.7)
[   +9 ms] I/flutter ( 4616): Offset(228.7, 387.3)
[   +7 ms] I/flutter ( 4616): Offset(238.2, 386.2)
[   +8 ms] I/flutter ( 4616): Offset(245.8, 385.1)
[   +8 ms] I/flutter ( 4616): Offset(255.3, 384.0)
[   +8 ms] I/flutter ( 4616): Offset(262.5, 382.5)
[   +9 ms] I/flutter ( 4616): Offset(268.4, 381.5)
[   +8 ms] I/flutter ( 4616): Offset(273.5, 380.4)
[   +8 ms] I/flutter ( 4616): Offset(278.9, 378.9)
[   +9 ms] I/flutter ( 4616): Offset(283.6, 377.5)
[   +7 ms] I/flutter ( 4616): Offset(288.0, 376.0)
[   +8 ms] I/flutter ( 4616): Offset(292.7, 374.5)
[   +8 ms] I/flutter ( 4616): Offset(297.1, 373.1)
[   +8 ms] I/flutter ( 4616): Offset(301.8, 371.6)
[   +8 ms] I/flutter ( 4616): Offset(307.3, 370.5)
[   +8 ms] I/flutter ( 4616): Offset(312.7, 369.1)
[   +8 ms] I/flutter ( 4616): Offset(317.8, 368.0)
[   +9 ms] I/flutter ( 4616): Offset(322.9, 367.3)
[   +7 ms] I/flutter ( 4616): Offset(326.9, 366.5)
[   +9 ms] I/flutter ( 4616): Offset(330.5, 366.2)
[  +10 ms] I/flutter ( 4616): Offset(332.7, 365.5)
[   +7 ms] I/flutter ( 4616): Offset(334.9, 365.1)
[   +9 ms] I/flutter ( 4616): Offset(336.0, 365.1)
[   +6 ms] I/flutter ( 4616): Offset(336.4, 364.7)
[   +6 ms] I/flutter ( 4616): calling 911
[  +35 ms] I/flutter ( 4616): Offset(336.7, 364.7)

```

</details>

<details>
  <summary>flutter doctor -v</summary>
```console
[✓] Flutter (Channel master, v1.18.0-5.0.pre.46, on Mac OS X 10.15.4 19E266, locale en-EE)
    • Flutter version 1.18.0-5.0.pre.46 at /Users/nevercode/development/flutter_master
    • Framework revision 0d459f2326 (7 hours ago), 2020-04-08 21:32:25 -0700
    • Engine revision 5b4b1f33c6
    • Dart version 2.8.0 (build 2.8.0-dev.20.0 dcdc71d763)


[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/nevercode/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.9.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 45.0.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] VS Code
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.9.1

[✓] Connected device (3 available)
    • Pixel 3a   • 965AY0WP5C • android-arm64  • Android 10 (API 29)
    • Chrome     • chrome     • web-javascript • Google Chrome 80.0.3987.163
    • Web Server • web-server • web-javascript • Flutter Tools

• No issues found!
```
</details>

<details>
  <summary>code sample</summary>

```dart
import 'package:flutter/material.dart';
import 'package:url_launcher/url_launcher.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) => MaterialApp(
        home: MyHomePage(),
      );
}

class MyHomePage extends StatefulWidget {
  @override
  _MyHomePageState createState() => _MyHomePageState();
}

class _MyHomePageState extends State<MyHomePage> {
  
  @override
  void initState() { 
    super.initState();
    _delayedPhoneCall();
  }

  void _delayedPhoneCall() async {
    await Future.delayed(const Duration(seconds: 5),);
    print('calling 911');
    launch('tel:911');
  }

  @override
  Widget build(BuildContext context) => Scaffold(
    body: GestureDetector(
      onPanUpdate: (details) => print(details.globalPosition),
      child: Container(
        color: Colors.orange,
      ),
    ),
  );
}
```

</details>

May I close the issue?
Thank you

**Comment by miguelcmedeiros** [comment]
@iapicca 
Thanks for re-checking and for confirming that it's solved!!
